spec.emu

```diff
@@ -49,6 +41,17 @@
   <p>This proposal would ideally use the `with` keyword to denote assertions, but there are existing implementations based on the previous version of the proposal using the `assert` keyword. Due to potential web compatibility risks, the proposal still includes `assert` marked as <emu-xref href="#sec-conformance">normative optional</emu-xref> and <emu-xref href="#sec-conformance">legacy</emu-xref> in sections <emu-xref href="#sec-evaluate-import-call"></emu-xref> and <emu-xref href="#sec-imports"></emu-xref>. Usage of the old syntax is discouraged, and it removal is being investigated.</p>
 </emu-intro>
 
+<emu-clause id="sec-conformance" number="2">
+  <h1>Conformance</h1>
+
+  <p><ins>A conforming implementation of ECMAScript should not implement <dfn>Deprecated</dfn> subclauses. All of the language features and behaviours specified within Deprecated subclauses have one or more undesirable characteristics. However, their usage in existing applications prevents their removal from this specification. These features are not considered part of the core ECMAScript language. Programmers should not use or assume the existence of these features and behaviours when writing new ECMAScript code.</ins></p>
```

Comment:
i love this as-is, but, this implies that v8 is nonconforming, and I don't think that's something we want.

Reply:
It's a _should_ and not a _must_. However, since this spec doesn't refer to what should/must/... mean, maybe something like this to make it clearer?

> A conforming implementation of ECMAScript should not implement Deprecated subclauses, <ins>unless necessary due to compatibility with existing applications that run in such implementation</ins>. All of the language features and behaviours specified within Deprecated subclauses have one or more undesirable characteristics. However, their usage in existing applications currently prevents their removal from this specification. These features are not considered part of the core ECMAScript language. Programmers should not use or assume the existence of these features and behaviours when writing new ECMAScript code.

Reply:
I pushed a commit which should make it clearer.